INSTALL.md

```diff
@@ -10,36 +10,36 @@ To use JavaMOP you need JRE, AJC and RV-Monitor.
 
 1. JRE
 
- Java Runtime Environment version 7 or higher (http://java.com/en/download/index.jsp)
+ Java Development Kit (JDK) version 7.0.0 or higher 
+ (http://www.oracle.com/technetwork/java/javase/downloads/index.html)
 
 * To make sure java is installed properly, run `java -version` from a
-  terminal.
+  terminal. Also if you need to use agent in JavaMOP, also make sure
+  `jar` is in your `PATH`.
 
 2. AJC
 
  AspectJ Compiler version 1.8.1 or higher
  (http://www.eclipse.org/aspectj/downloads.php)
 
- * Make sure that `<Path-to-AspectJ_HOME>/bin` is in your `PATH` by
+ * Make sure that `<AspectJ_HOME>/bin` is in your `PATH` by
    calling `ajc` in a terminal.
 
- * Also ensure that `<Path-to-AspectJ_HOME>/lib/aspectjrt.jar`,
-   `<Path-to-AspectJ_HOME>/lib/aspectjweaver.jar` and
-   `<Path-to-AspectJ_HOME>/lib/aspectjtools.jar` are in your
+ * Also ensure that `<AspectJ_HOME>/lib/aspectjrt.jar`,
+   `<AspectJ_HOME>/lib/aspectjweaver.jar` and
+   `<AspectJ_HOME>/lib/aspectjtools.jar` are in your
    CLASSPATH.
```

Comment:
Hi,

I&#39;d like to add you to my professional network on LinkedIn.

- Qingzhou

Accept: https://www.linkedin.com/blink?simpleRedirect=3gQejgTd3ANc3APcz0VdPkTejkZh4BKrSBQonhFtCVF9CllkQYVoSVPfnBBiShBsC5EsOpQsSlRpRZBt6BSrCAZqSkConhzbmlQqnpKqiRQsSlRpORIrmkZpSVFqSdxsDgCpnhFtCV9pSlipn9Mfm4CojoJqTtAcCNFdCAJsSxRdC5TbjRBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&msgID=I5975790255276589056_500&markAsRead=


You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/blink?simpleRedirect=64SbmJTp39IqjpFbndEtjpxtOQZp6BB9A5Udz1nrDdAm6xdl5ZGkzBPp4J4oTd2lSxobkBjuzdLs5pimmxEl6t8tTxmqPtkkTFnrDtThBpPbktWkS98iCROskJIgzlvsAF7tBhGqP1okB1Wk45KkBZgenkQt4gUi59Ac48RgmN4enx4nSlcbkVbd50Qp5tkm6JMe718rQ9JjzpTh7dFd5pnqARQj756dnllgk51gmtAc7tKmlp1iTt6kk4Zp6BLr2pQsSlRpRZBt6BSrCAZqSkCkjoPp4l7q5p6sCR6kk4ZrClHrRhAqmQCrDlIfngCojoJqTtAcCNFdCAJsSxRdC5TbjRBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&amp;msgID=I5975790255276589056_500&amp;markAsRead= Learn why we included this at the following link: https://www.linkedin.com/blink?simpleRedirect=0Ue3sQfmh9pmNzqnhOoipQsSlRpRZBt6BSrCAZqSkCr79lpmdFtD9BkT9BrmZQsTlzfm4CojoJqTtAcCNFdCAJsSxRdC5TbjRBfP9SbSkLrmZzbCVFp6lHrCBIbDtTtOYLeDdMt7hE&msgID=I5975790255276589056_500&markAsRead=
&copy; 2014, LinkedIn Corporation. 2029 Stierlin Ct. Mountain View, CA 94043, USA